assets/sass/_pulumi-up.scss

```diff
@@ -64,6 +64,12 @@ $orange900: theme("colors.orange.900");
     }
 }
 
+.orange-neon {
+    border-radius: 0.25rem;
+    border-color: #e17d47;
```

Comment:
You can assign a variable that pulls in the value of the config, like some of the other variables at the top of the file. Example:
```
$orange600: theme('colors.orange.600');
// other stuff
.orange-neon {
    // ... styles
    border-color: $orange600;
}
```